### PR TITLE
Removes Box Station

### PR DESCRIPTION
**Pull request by actioninja**
# Hi, Reddit!
https://www.reddit.com/r/SS13/comments/g9hag9/to_all_who_play_on_tg_codebase_please_take_a_look/
This PR is now locked due to outside brigading.
If you are interested in actual discussion of this topic, please go to the forum thread: https://tgstation13.org/phpBB/viewtopic.php?f=5&t=26554

## About The Pull Request

Removes Box Station Entirely

## Why It's Good For The Game

Box Station has been antiquated and deserved removal for a very long time coming now.

Recent attempts to update particularly bad parts of Box (powernet, pipenet, etc.) fail to address the single largest issue with box. Meta was so heavily based on Box that Meta is effectively just a better version of Box.

The way I see it Box Station will never match either the popularity or the quality of MetaStation due to fundamental problems with the layout and design.
On top of that, they are such similar of maps that maintaining both at the same time feels like it adds very little. Say what you will about maps that aren't Box or Meta, they are the two most fundamentally similar maps in the game. Even Delta, which is yet another Box derivative, doesn't come off nearly as similar to Box.

There is a push to reduce the number of active maps and there is zero reason to keep around a bunch of maps that are extremely similar. The whole advantage of running multiple maps is variety, and having only Box, Meta, Delta, and the flavor of the month fourth map that everyone hates because it's not one of the three extremely similar maps is a colossal waste of time and effort for everybody involved.

## Changelog
:cl:
del: Box Station
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
pretty boxed and pilled

**Comment by Zytolg** [comment]
And thus the circle of fate begins. Having passed the crown to Meta long ago, Box now falls. While the current king may reign supreme, already, it's successor gains momentum in the form of the **D**. Delta will one day see an offspring of it's own, and that will be the day Meta falls from the throne. So it is written, so it shall be.

**Comment by wesoda25** [comment]
I mean we're getting ice box so what does this really do?

**Comment by Jungling** [comment]
When you going to optimize the codebase so it stops lagging and make it so delta/meta don't hard crash a server after running for an hour? Just wondering when that'll happen, since you want to remove box. Just wondering, Curious really. Downvoted btw.

**Comment by optimumtact** [comment]
see you later alt

**Comment by kriskog** [comment]
Long overdue

**Comment by CthulhuOnIce** [comment]
😢

**Comment by SynnGraffkin** [comment]
Why not try to apply the love and care that goes into Box on other servers?

**Comment by optimumtact** [comment]
I think anyone with two molecules to rub together to make a brain could see that the "love and effort" goes into maps that are not box on this codebase.

**Comment by SynnGraffkin** [comment]
Maybe look into features other servers have on Box that makes it stand out compared to Meta / Delta. Bring the AI back under the bridge, randomize Bar / Engine layouts, stuff like that. Something that makes Box stand out more.

**Comment by optimumtact** [comment]
Again, it's pretty clear you are asking vegans to build you a ham burger.

**Comment by SynnGraffkin** [comment]
I guess it is more so trying to convince a TG maintainer to improve, not remove.

**Comment by wesoda25** [comment]
> I think anyone with two molecules to rub together to make a brain could see that the "love and effort" goes into maps that are not box on this codebase.
> 
> so this is rather like asking a vegetarian why they don't just try raising the meat themself

I mean... I'd say its been getting SOME "love and effort" as of late, or at least the latter
#50704
#50430
#50108 (small but effort i guess)

**Comment by dee-dubya** [comment]
zoom zoom

**Comment by dinnerwoerror** [comment]
hold on, lemme do a Paradox dice roll to determine if the round is gonna grind to a halt at the 25 minute mark because the powernet decided to not work

**Comment by MarkSuckerberg** [comment]
I wanna say something but I don't want to be blocked, so I'm just going to say that I have no particular feelings about this pull request other than it's sad that it says that it should default to box despite the fact it's now set to meta

**Comment by triplezeta** [comment]
I'll miss you, box. Rest in peace, and rest well. 
salute

**Comment by optimumtact** [comment]
don't mind me, just removing some boomers

**Comment by MrDoomBringer** [comment]
I assume we're still getting icebox station?

**Comment by actioninja** [comment]
This PR is now locked due to an outside brigade by redditors. Please go to the forums if you wish to discuss the matter.
https://tgstation13.org/phpBB/viewtopic.php?f=5&t=26554

File: code/datums/map_config.dm [hunk 1]
```diff
@@ -14,9 +14,9 @@
 	var/votable = FALSE
 
 	// Config actually from the JSON - should default to Box
```
Comment (duncathan) [inline code/datums/map_config.dm:16]: comment needs to be updated

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
@MrDoomBringer 
Now here's a little lesson in trickery

**Comment by optimumtact** [comment]
The people of sybil have voted, plus I don't think it's likely box is going to be removed any time soon

Donut and Kilo station are being removed to make way for new maps though.

We could potentially do a longer poll about replacing box for icebox instead, and I would encourage you to ask me to set that up

**Pull request closed without merging**